### PR TITLE
Fix path to install files

### PR DESCRIPTION
**Pull request by markwinter**
Get KFServing YAML from Github Releases so it's not dependent on script execution path

### CODE REVIEW

File: hack/quick_install.sh [hunk 1]
```diff
@@ -56,6 +56,6 @@ kubectl wait --for=condition=available --timeout=600s deployment/cert-manager-we
 cd ..
 # Install KFServing
 # Retry inorder to handle that it may take a minute or so for the TLS assets required for the webhook to function to be provisioned
-for i in 1 2 3 4 5 ; do kubectl apply -f install/${KFSERVING_VERSION}/kfserving.yaml && break || sleep 15; done
+for i in 1 2 3 4 5 ; do kubectl apply -f ../install/${KFSERVING_VERSION}/kfserving.yaml && break || sleep 15; done
```
Comment (yuzisun) [inline hack/quick_install.sh:59]: This highly depends on where you run the script, I'd suggest to change to use the release yaml `https://github.com/kubeflow/kfserving/releases/download/$KFSERVING_VERSION/kfserving.yaml`

File: hack/quick_install.sh [hunk 2]
```diff
@@ -56,6 +56,6 @@ kubectl wait --for=condition=available --timeout=600s deployment/cert-manager-we
 cd ..
 # Install KFServing
 # Retry inorder to handle that it may take a minute or so for the TLS assets required for the webhook to function to be provisioned
-for i in 1 2 3 4 5 ; do kubectl apply -f https://github.com/kubeflow/kfserving/releases/download/$KFSERVING_VERSION/kfserving.yaml && break || sleep 15; done
+for i in 1 2 3 4 5 ; do kubectl apply -f https://github.com/kubeflow/kfserving/releases/download/${KFSERVING_VERSION}/kfserving.yaml && break || sleep 15; done
```
Comment (pugangxa) [inline hack/quick_install.sh:59]: What's this change for?

Comment (markwinter) [reply hack/quick_install.sh:59]: Consistency with the rest of the script

**Comment by yuzisun** [comment]
Please help move the PR to https://github.com/kserve/kserve, for more information please see [kubeflow mailing thread](https://mail.google.com/mail/u/2/?pli=1#search/kfserving/FMfcgzGkbDbSkjqCfZBscVwsHQTxNdnC), thanks.

**Pull request closed without merging**